src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncSpec.cs

```diff
@@ -299,6 +300,9 @@ public void A_Flow_with_SelectAsync_must_handle_cancel_properly()
         [Fact]
         public void A_Flow_with_SelectAsync_must_not_run_more_futures_than_configured()
         {
+            if (IsMono) // TODO: 9/14/2016: Mono 4.4.2 blows up the XUnit test runner upon calling Thread.Interrupt below (@Aaronontheweb)
```

Comment:
Is this issue noticed anywhere? If not, please create a ticket for it. This is not a good practice.